pydis_site/apps/api/viewsets/bot/offensive_message.py

```diff
@@ -46,6 +47,16 @@ class OffensiveMessageViewSet(
     - 201: returned on success
     - 400: if the body format is invalid
 
+    ### PATCH /bot/offensive-messages/<id:int>
```

Comment:
I just have one question, do we want to allow patching of all fields, or just the deleting date ?

Reply:
This (currently) allows patching all fields. I can make the other fields write-once, but that will be a bit more work.

Reply:
Sure, I was just wondering what we wanted to have. If we only want to allow date modifications, then we might block the others as well.